pyccel/decorators.py

```diff
@@ -26,8 +26,8 @@
 def lambdify(f):
 
     args = f.__code__.co_varnames
-    from sympy import symbols
-    args = symbols(args)
+    from pyccel.ast.internals import Symbol
+    args = Symbol(args)
     expr = f(*args)
     def wrapper(*vals):
         return  expr.subs(zip(args,vals)).doit()
```

Comment:
@saidctb Can you provide more information (or a docstring) about lambdify?
Does it handle sympy objects or pyccel objects?

Reply:
here it must stay with sympy

Reply:
here we are trying to mimic the lambdify of sympy